### PR TITLE
[Clear Fluid API]Move paddle.fluid.layers.tensor.create_parameter

### PR DESCRIPTION
**Pull request by cyber-pioneer**
### PR types
Others

### PR changes
APIs

### Describe
Move paddle.fluid.layers.tensor.create_parameter to paddle.tensor.creation.create_parameter.

### CODE REVIEW

**Comment by cyber-pioneer** [comment]
官网doc预览正常：
http://10.136.157.23:8090/documentation/docs/en/develop/api/paddle/create_parameter_en.html

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

File: python/paddle/tensor/creation.py [hunk 1]
```diff
@@ -67,6 +70,88 @@ def _real_to_complex_dtype(dtype):
         return dtype
 
 
+def create_parameter(
+    shape, dtype, name=None, attr=None, is_bias=False, default_initializer=None
+):
+    """
+        :api_attr: Static Graph
```
Comment (Ligoml) [inline python/paddle/tensor/creation.py]: 删掉

File: python/paddle/tensor/creation.py [hunk 2]
```diff
@@ -67,6 +70,88 @@ def _real_to_complex_dtype(dtype):
...
+
+    This function creates a parameter. The parameter is a learnable variable, which can have
+    gradient, and can be optimized.
+
+    NOTE: this is a very low-level API. This API is useful when you create
```
Comment (Ligoml) [inline python/paddle/tensor/creation.py]: note的格式不对，辛苦查看一下写作规范

**Review by Ligoml — APPROVED** [review]
LGTM for docs

File: python/paddle/tensor/creation.py (hunk 1 above)
Comment (cyber-pioneer) [reply python/paddle/tensor/creation.py]: done

File: python/paddle/tensor/creation.py (hunk 2 above)
Comment (cyber-pioneer) [reply python/paddle/tensor/creation.py]: done

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**